resources/views/settings/cards/webhooks.blade.php

```diff
@@ -0,0 +1,35 @@
+@php use Carbon\Carbon; @endphp
+<div class="card mt-3">
+    <div class="card-header">{{ __('settings.title-webhooks') }}</div>
+    <div class="card-body">
+        @if(count($webhooks) == 0)
+            <p class="text-danger">{{__('settings.no-access-from-external-applications')}}</p>
+        @else
+            <table class="table table-responsive">
+                <thead>
+                    <tr>
+                        <th>{{ __('settings.client-name') }}</th>
+                        <th>{{ __('settings.created') }}</th>
+                    </tr>
```

Comment:
Would it make sense to show the user to which events a webhook responds? Just thinking out lout here.

Reply:
It definitely would, good point